code/modules/projectiles/projectile/special.dm

```diff
@@ -119,6 +119,10 @@
 	flag = "bomb"
 	range = 3
 
+/obj/item/projectile/kinetic/upgraded
+	damage = 12

```

Comment:
4000 mining points of material sent to centcomm could buy ten laser rifles for thi-

wait a minute, lasers are 20 damage. 12 is fire extinguisher tier.